bin/program.py

```diff
@@ -275,7 +275,7 @@ def _get_language(self, bar: ProgressBar):
                 'mainclass': mainclass,
                 'Mainclass': mainclass[0].upper() + mainclass[1:],
                 # Memory limit in MB.
-                'memlim': (get_memory_limit() or 1024),
+                'memlim': 2048,  # TODO use validation_memory?
```

Comment:
It could make sense to use `validation_memory` here (assuming you refer to the one in the [Limits section](https://www.kattis.com/problem-package-format/spec/2023-07-draft.html#limits)), but `Program` is also used for generators and visualizers, not only validators, so I'm not 100% sure if that would be the correct thing to do :thinking: On the other hand, the spec only defines three types of "memory" and generators/visualizers are not mentioned in these memory limits, so we can do what we want? :joy: